lbz/authz/__init__.py

```diff
@@ -1,4 +0,0 @@
-# flake8: noqa
```

Comment:
You cannot just remove `__init__.py` module. That makes `authz` a normal directory (it should be a Python package) and probably, it won't be found during building the release for `lbz` (run `python setup.py sdist` to see that).

Please, describe here the problem you wanted to avoid, maybe there is a better idea to solve that 😉

Reply:
I checked it more deeply and it looks like the bug has been introduced in #39. A newly created package hasn't been mentioned in the `lbz` [packages](https://github.com/pdyba/lambdalizator/blob/master/setup.py#L9) - it means v0.3.16 is broken because it does not contain `authz` at all.

But still, we cannot remove `__init__.py` from here.